### ISSUE TITLE
Increase base font size

### ISSUE THREAD

**Issue by bronzehedwick** [bronzehedwick/chrisdeluca#72]
The base font size should be around 20px, but judge by eye. This will make reading easier for everyone. See: https://blog.marvelapp.com/body-text-small/
